### PR TITLE
fix #600 : Removed unused methods and variables

### LINKED ISSUE #600: Unused Methods and Variables
There are some unused methods and variables.Some of them are assigned but were never used and can be removed safely. Below is a list of the class name with respective methods and variables

1) AppListActivity--getCheckedIdObjects method is never used

2) BearingActivity--mBearing variable is never used(assigned value mBearing = String.format("%.3f", degrees)

3) FormEntryActivity--saveAnswer method is never used

4) GeoPointMapActivity--mRefreshLocation never used
                                         mZoomed(assigned some values but never used)
                                         mLocationAccuracy (assigned some values but never used)

5) GeoPointOsmMapActivity-- mHandler , sharedPreferences , basemap  never used
                                                mZoomed (assigned some values but never used) 
                                                mLocationAccuracy (assigned some values but never used)

6) GeoShapeGoogleMapActivity-- sharedPreferences ,basemap,gmapSettings ,polygon_button never used
                                                   initZoom(assigned some values but never used) 
                                                   latLngsArray (assigned some values but never used)
                                                   data_loaded (assigned some values but never used)
                                                   clear_button_test (assigned some values but never used)

7) GeoShapeOsmMapActivity--polygon_connection(assigned some values but never used)
                                                 sharedPreferences , layerStatus , selected_layer  never used

8) GeoTraceGoogleMapActivity-- polygon_button , progress , sharedPreferences ,  gmapSettings 
                                                     basemap , latLngsArray  never used
                                                     inital_location_found (assigned some values but never used) 
                                                     clear_button_test (assigned some values but never used)
                                                     initZoom(assigned some values but never used)

9) GeoTraceOsmMapActivity-- sharedPreferences(assigned some values but never used)
                                                 inital_location_found(assigned some values but never used)

### PR DESCRIPTION
**Pull request by supreme96**
The methods and fields in different files that were mentioned in the issue #600 that were present in the code but were never used have been removed. Each of the files that were refactored have their changes summarized in individual commits and each refactored file was followed by some manual testing of the application. 

However, further testing is recommended.

### CODE REVIEW

File: collect_app/build.gradle [hunk 1]
```diff
@@ -63,7 +63,7 @@ android {
     defaultConfig {
         applicationId("org.odk.collect.android")
         minSdkVersion(16)
-        targetSdkVersion(22)
+        targetSdkVersion(25)
```
Comment (shobhitagarwal1612) [inline collect_app/build.gradle]: Updating the targetSdkVersion was not the concern of the issue. Please try not to make changes more other than the scope of the issue

**Comment by royrivnam** [comment]
i guess there are some more classes with unused variables and methods..so can i remove the remaining unused variables and methods..

File: collect_app/build.gradle (hunk 1 above)
Comment (supreme96) [reply collect_app/build.gradle]: @shobhitagarwal1612 Is there a way to correct this now?

Comment (shobhitagarwal1612) [reply collect_app/build.gradle]: Revert back the changes in another commit. Also, there are some merge conflicts. Could you please resolve them as well?

**Comment by nribeka** [comment]
I think you can run the Android Studio's analysis and go through the result of the analysis right?

**Comment by supreme96** [comment]
fix #600 : Refactor fully complete. No merge conflicts.

**Pull request merged**